### PR TITLE
[#94939334] Create dedicated gandalf playbook

### PR DESCRIPTION
**Pull request by actionjack**
[Manage all Ansible dependencies from Ansible Galaxy](https://www.pivotaltracker.com/story/show/94939334)

**What**

This PR is to split out the gandalf playbook from our [tsuru-ansible](https://github.com/alphagov/tsuru-ansible) repository into it's own dedicated Github repository.

**How to review this PR**

This PR should be reviewed in the following context:
1. I want to move the existing gandalf playbook from out of the role-static directory,
2. I no longer want the other playbooks so I am deleting the roles-static directory,
3. I want to inform interested parties of the license this software and the terms of it's re-use,
4. I want to be able to insert this playbook into ansible galaxy at a future date.

**Who should review this PR**
- Anyone in the core team.
- I paired with @dhilton with this one so he can't merge it :tongue:

### CODE REVIEW

**Comment by annashipman** [comment]
We talked about using `git filter-branch` to split the repo where this came from so we don't lose all the history of what we've previously done to this module. Is there a reason you decided not to do that?

**Comment by actionjack** [comment]
I did use `git-filter-branch` have you checked [master](https://github.com/alphagov/ansible-playbook-gandalf/commits/master) ? What I did was:
- Created a new master branch using on `git filter-branch`
- Branched off that point in time

**Comment by annashipman** [comment]
You're absolutely right, I did not look at master :) That'll teach me for looking quickly on my way into work. Sorry!

In that case, this is great. One tiny comment.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+
+    - hosts: gandalf-server
+      roles:
+         - { role: gandalf, gandalf_port: 8080, gandalf_git_template_dir: /home/git/bare-template }
+
+License
+-------
+
+See `LICENSE` file.
```
Comment (annashipman) [inline README.md:35]: You could link to the licence file here.

**Comment by annashipman** [comment]
Not enough to block it though, maybe just change it sometime in future.

**Pull request merged**